config/routes.rb

```diff
@@ -3,12 +3,18 @@
   root 'sessions#new'
 
   get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
+  post 'login', to: 'sessions#create', as: 'post_login'
```

Comment:
Ah this! I needed to use a `redirect_to` in order to get make a post to the login page.
I looked everywhere and couldn't find a way to specify a method within that rails function, so I created a prefix for it so I can reference that instead and send information through the prefix.